README.md

```diff
@@ -1 +1,10 @@
-# azure-storage-samples-for-python
\ No newline at end of file
+#Client Side Encryption to CPK-N Migration
+#####Various Migration Samples From Client Side Encryption to Server Side Encryption
+
+##General Info
+These samples begin with an optional data creation process to create an example container and blob with client side encryption. The samples then migrate the specified blob to a specific type of server side encryption and uploads the server side encrypted blob in the same container.
+
+##General Setup Requirements
+All samples require users to enter their storage credentials in the config.py file. For more details on setup, refer to the README of the specific sample used.
+
+The migration.py file can also be ran without running the exampleDataCreator.py file, however all components of the setup must be in place: container, blob uploaded with client side encryption, encryption scope where applicable, keyvault keys where applicable, keyvault secret where applicable, etc. All of these values can be inputted in the config.py file for each scenario.
```

Comment:
>config [](start = 343, length = 6)

also here. you can use Ctrl+Shift+r in Pycharm to find all config.py and replace it one by one or do replace all